app/Http/Controllers/SubmissionsController.php

```diff
@@ -20,6 +21,18 @@ public function store(Request $request)
         return response()->json(['status' => 'success', 'message' => 'Talk Submitted']);
     }
 
+    public function update(Request $request)
```

Comment:
What if this was instead AcceptancesController@create()?

Reply:
I can do that. I was being a bit lazy here. Wanted to get the data model right first.